### PR TITLE
Update/Fix README

### PR DESCRIPTION
**Pull request by ignxdank**
Update README to v2.0.0, add API key changes information, change vanity to static invite as vanity URL may not remain consistently and fix some typos & grammar

### CODE REVIEW

**Comment by Cryptizism** [comment]
LGTM but @Chit132 ik you wanted somethings so you look over it

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# <img src="https://user-images.githubusercontent.com/61895718/111565782-5ced2900-8772-11eb-9c43-c8801fc2a1a8.png" width="32px"> 𝗔𝗯𝘆𝘀𝘀 𝗢𝘃𝗲𝗿𝗹𝗮𝘆 𝘃𝟭.𝟳.𝟬
+# <img src="https://user-images.githubusercontent.com/61895718/111565782-5ced2900-8772-11eb-9c43-c8801fc2a1a8.png" width="32px"> 𝗔𝗯𝘆𝘀𝘀 𝗢𝘃𝗲𝗿𝗹𝗮𝘆 𝘃2.0.0
 
-![GitHub package.json version](https://img.shields.io/github/package-json/v/Chit132/abyss-overlay) ![GitHub all releases](https://img.shields.io/github/downloads/Chit132/abyss-overlay/total) [![Discord](https://img.shields.io/discord/822639649247592528?color=7289DA&label=Discord)](https://discord.gg/abyssoverlay)
+![GitHub package.json version](https://img.shields.io/github/package-json/v/Chit132/abyss-overlay) ![GitHub all releases](https://img.shields.io/github/downloads/Chit132/abyss-overlay/total) [![Discord](https://img.shields.io/discord/822639649247592528?color=7289DA&label=Discord)](https://discord.gg/eDnHTT3aBS)
```
Comment (Cryptizism) [inline README.md:3]: Can you please change the shields to this new version as it displays members not only online users per chit's request

https://dcbadge.vercel.app/api/server/eDnHTT3aBS?style=plastic

**Review by Cryptizism — CHANGES_REQUESTED** [review]
Add the following to features:
- auto detecting logs
- no api key or authentication
- popup stats

Amend the badge as like: https://github.com/Chit132/abyss-overlay/pull/42#discussion_r1285863334

Chit also wants to have a new image so wait for chit to give that

**Comment by ignxdank** [comment]
The badge image has been updated to display the total member count as opposed to just online and been committed to this PR, now just waiting on the image before committing the requested changes.

**Review by Cryptizism — APPROVED** [review]

**Pull request merged**